frontend/src/components/UI/Layouts/index.md

```diff
@@ -0,0 +1,7 @@
+# LAYOUTS
```

Comment:
could try this new pattern to document what is inside each directory a la `Nuxt.js` project

Reply:
https://nuxt.com/docs/guide/directory-structure/layouts